1_create_database.md

```diff
@@ -0,0 +1,88 @@
+### 1. Tạo database như hình trên (sau khi tạo thì import database)
+- Create database
+```mysql
+  CREATE DATABASE `train_mysql` DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci;
+```
+- Create table `user`
+```mysql
+CREATE TABLE `user` (
+  `id` INT(11) NOT NULL AUTO_INCREMENT,
+  `full_name` VARCHAR(255),
+  `email` VARCHAR(255),
+  `rank` TINYINT(4),
+  `is_active` TINYINT(1),
+  `created_at` TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  PRIMARY KEY(`id`)
+);
+```
+- Create table `category`
+```mysql
+CREATE TABLE `category` (
```

Comment:
lệnh tạo table của em cũng đúng nhưng nên dùng
`CREATE TABLE IF NOT EXISTS category `
thì hay hơn, em đổi lại hết tất cả đi